routing/src/fib/fibtype.rs

```diff
@@ -219,11 +219,17 @@ impl Fib {
     pub fn lpm_with_prefix(&self, target: &IpAddr) -> (Prefix, &FibGroup) {
         match target {
             IpAddr::V4(a) => {
-                let (prefix, group) = self.routesv4.lookup(a);
+                let (prefix, group) = self
+                    .routesv4
+                    .get_lpm(&Ipv4Prefix::from(*a))
+                    .unwrap_or_else(|| unreachable!());
```

Comment:
How can you be sure you never reach this? My understanding is that `.get_lpm()` can return `None`, contrarily to the `iptrie` lookup.